### ISSUE TITLE
Friends not Targeted by Killaura

### ISSUE THREAD

**Issue by VincentCCandela** [ImpactDevelopment/ImpactIssues#2127]
**Describe the bug**
When I turn on killuara and set it so that it only targets friends, it does not target friends. Players is off as is all the other targets. Yes, I do know that they were added as friends. I added them with the middle-click friend mod as well as adding them using commands. 

**To Reproduce**
1. Add people as friends
2. Turn on killaura targeting only friends
3. watch killuara not work

**Expected behavior**
I expect killaura to target friends

**Screenshots**
Not necessary

- Impact Version: 4.8.3 for minecraft 1.14.4
- Using Optifine? No
- Using LiteLoader? No
- Enabled mods: MiddleClickFriend, Chams Wallhack, Anti Sound Lag,
No Fall Bucket, Storage ESP, Trajectories, Esp Shader, anti vanish, nametags, anti blind, autofish, sprint
- Operating System: Ubuntu 18.04

</details>

**Additional context**

**Comment by VincentCCandela** [comment]
Basically, for the killaura mod, I want to be able to enable the friends option without enabling the players option.

**Comment by S-B99** [comment]
Friends are players though.

**Comment by VincentCCandela** [comment]
No, but thats the exact problem. I don't want killaura to hit everybody. I only want it to hit friends. Because if killaura hits everybody, then it will trigger the anti cheat.

**Comment by S-B99** [comment]
So create a suggestion for a friends only mode

**Comment by LeafHacker** [comment]
> So create a suggestion for a friends only mode

That's what this issue is lol

**Comment by S-B99** [comment]
The issue was a bug

**Comment by S-B99** [comment]
Not a suggestion

**Comment by LeafHacker** [comment]
That's overly picky. It's also subjective as to whether this is a bug or a feature. IMO not being able to target friends only is a _design flaw_ rather than a _code error_, but still a bug.
